### PR TITLE
Create Chanel component

### PR DESCRIPTION
**Pull request by maksim-karatkevich**

### CODE REVIEW

File: src/app/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Link, Route, Switch } from 'react-router-dom';
+import React from 'react';
+import ChannelsList from '../features/Chanels/ChannelsList';
+import Channel from '../features/Chanels/Channel';
+import 'antd/dist/antd.css';
```
Comment (Roma36) [inline src/app/routes/index.js:5]: please, move this to app.js

File: src/app/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const Routes = () => {
+  return (
+    <div>
+      <nav className="navbar navbar-dark bg-primary d-block">
```
Comment (Roma36) [inline src/app/routes/index.js:10]: Consider creating Header (or Navigation) component and moving it to app.js. The idea is that nav bar may be extended with other control buttons, logo, etc.

File: src/app/features/Chanels/ChannelsList.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  return (
+    <div className="m-4">
+      <AddChannelForm />
+      <Row>
+        {mockData.map((item, index) => {
+          return (
+            <Link
+              key={`${item}`}
+              to={{ pathname: `${url}/${index}`, state: item }}
```
Comment (Roma36) [inline src/app/features/Chanels/ChannelsList.js:31]: normally, you don't need to pass the item via Route state. For now it's ok, but later when data will be handled by redux, the Link will look like this:  <Link to={`${url}/${index}`}

File: src/app/features/Chanels/ChannelsList.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  const { url } = useRouteMatch();
+
+  return (
+    <div className="m-4">
+      <AddChannelForm />
+      <Row>
+        {mockData.map((item, index) => {
+          return (
+            <Link
+              key={`${item}`}
```
Comment (Roma36) [inline src/app/features/Chanels/ChannelsList.js:30]: Check errors in console. key should be unique. So, as soon as channel name is unique - use it as key.
```suggestion
              key={item.name}
```

File: .gitignore [hunk 5]
```diff
@@ -26,6 +26,7 @@ bin-release/
 # testing
 /coverage
 
+
```
Comment (Roma36) [inline .gitignore:29]: why empty line? :)

File: src/app/routes/index.js (hunk 1 above)
Comment (maksim-karatkevich) [reply src/app/routes/index.js:5]: imported

**Pull request merged**